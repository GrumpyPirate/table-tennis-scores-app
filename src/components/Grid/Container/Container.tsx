import { rem } from 'polished';
import styled from 'styled-components';

import styleConfig from 'config/styles';

const Container = styled.div`
  width: 100%;
  padding-right: ${rem(styleConfig.grid.gutterWidths.mobile / 2)};
  padding-left: ${rem(styleConfig.grid.gutterWidths.mobile / 2)};

  @media ${styleConfig.mediaQueries.tablet} {
    padding-right: ${rem(styleConfig.grid.gutterWidths.tablet / 2)};
    padding-left: ${rem(styleConfig.grid.gutterWidths.tablet / 2)};
  }
`;

export default Container;
