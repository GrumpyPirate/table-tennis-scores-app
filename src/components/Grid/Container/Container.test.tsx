import { render } from '@testing-library/react';
import React from 'react';

import Container from './Container';

describe('Components/Grid/Container', () => {
  test('should render as expected, without crashing', () => {
    const { container } = render(
      <Container>
        <div>Some content</div>
      </Container>
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
