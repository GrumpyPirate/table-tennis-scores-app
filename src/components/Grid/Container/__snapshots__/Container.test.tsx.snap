// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Grid/Container should render as expected, without crashing 1`] = `
<div
  class="sc-bdfBwQ gvxfZJ"
>
  <div>
    Some content
  </div>
</div>
`;
